docs/static/jvm.asciidoc

```diff
@@ -59,3 +59,63 @@ a tarball.
 install the correct startup method (SysV init scripts, Upstart, or systemd). If
 {ls} is unable to find the `JAVA_HOME` environment variable during package
 installation, you may get an error message, and {ls} will not start properly.
+
+[float]
+[[jdk15-upgrade]]
+==== Using JDK 15
+
+{ls} supports JDK 15, but you need to update settings in `jvm.options` and
+`log4j2.properties` if: 
+
+* you are upgrading from  {ls} 7.10.x (or earlier) to 7.12 or later, and 
```

Comment:
```suggestion
* you are upgrading from  {ls} 7.11.x (or earlier) to 7.12 or later, and 
```

Reply:
Yes!  That's an important change. Thank you.

Reply:
Fixed